src/test/java/com/wikia/webdriver/testcases/visualeditor/VEVideoTests.java

```diff
@@ -0,0 +1,176 @@
+  @Execute(asUser = User.USER)
+  public void VEAddExternalVideoTests_003_AddExistingVid() {
+    VisualEditorPageObject ve =
+        new VisualEditorPageObject(driver).openVEOnArticle(wikiURL, PageContent.ARTICLE_NAME_PREFIX
+                                                                    + DateTime.now().getMillis());
+    ve.verifyVEToolBarPresent();
+    ve.verifyEditorSurfacePresent();
+    VisualEditorAddMediaDialog mediaDialog = ve.clickVideoButton();
+    mediaDialog = mediaDialog.searchMedia("y");
+    VisualEditorPageObject veNew = mediaDialog.addExistingMedia(2);
+    veNew.verifyVideos(2);
+    veNew.verifyVEToolBarPresent();
+    VisualEditorSaveChangesDialog save = veNew.clickPublishButton();
+    ArticlePageObject article = save.savePage();
+    article.verifyVEPublishComplete();
+  }
+
+
+  @Test(groups = {"VEVideo", "VEAddExternalVideoTests_004", "VEAddExistingVideo"})
+  @Execute(asUser = User.USER)
+  public void VEAddExternalVideoTests_004_RemoveVideoFromArticle() {
+    String articleName = PageContent.ARTICLE_NAME_PREFIX + DateTime.now().getMillis();
+    VisualEditorPageObject ve =
+        new VisualEditorPageObject(driver).openVEOnArticle(wikiURL, articleName);
+    ve.verifyVEToolBarPresent();
+    ve.verifyEditorSurfacePresent();
+    VisualEditorAddMediaDialog mediaDialog = ve.clickVideoButton();
+    VisualEditorPageObject veNew = mediaDialog.addMediaByURL(VideoContent.NON_PREMIUM_VIDEO_URL);
+    veNew.verifyVideos(1);
+    veNew.verifyVEToolBarPresent();
+    VisualEditorSaveChangesDialog save = veNew.clickPublishButton();
+    ArticlePageObject article = save.savePage();
+    article.verifyVEPublishComplete();
+
+    ve.openVEOnArticle(wikiURL, articleName);
+    ve.verifyVEToolBarPresent();
+    ve.verifyEditorSurfacePresent();
+    ve.selectMediaAndDelete();
+    ve.verifyNoVideo();
+    save = ve.clickPublishButton();
+    article = save.savePage();
+    article.verifyVEPublishComplete();
+  }
+
+  @Test(groups = {"VEVideo", "VEVideo_001", "VEVideoPreview"})
+  @Execute(asUser = User.USER_9)
+  public void VEVideoTests_001_previewVideo() {
+    String mediaTitle =
+        "Short film directed by Guy Ritchie starring David Beckham - H&M Spring 2013";
+    String providerName = "youtube";
+
+    String articleName = PageContent.ARTICLE_NAME_PREFIX + DateTime.now().getMillis();
+    VisualEditorPageObject ve =
+        new VisualEditorPageObject(driver).openVEOnArticle(wikiURL, articleName);
+    ve.verifyVEToolBarPresent();
+    ve.verifyEditorSurfacePresent();
+    VisualEditorAddMediaDialog mediaDialog = ve.clickVideoButton();
+    mediaDialog = mediaDialog.searchMedia(mediaTitle);
+    ve = mediaDialog.previewExistingMediaByTitle(mediaTitle);
+    ve.verifyPreviewVideoPlay(providerName);
+  }
+
+  @Test(groups = {"VEVideo", "VEVideo_002", "VEMediaResize"})
+  @Execute(asUser = User.USER_9)
+  public void VEVideoTests_002_resizeVideoWithHandle() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)